### PR TITLE
Generated new rails app

### PR DESCRIPTION
**Pull request by vsobotka**
I have created the most basic rails app.
This is what `rails` generates after invoking `rails new media-library`.

All you need to do to start the app is `cd media-library` and `docker-compose up`. Hopefully. Not sure if I configured Docker correctly. May not reflect changes/rebuild.

### CODE REVIEW

**Comment by grogy** [comment]
Thanks for you PR, @vsobotka .

I cannot now run this application, when I run `docker-compose up` then I see only warnings / erros. Can you try it, please?

**Comment by vsobotka** [comment]
Should work now.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+web:
+    build: .
+    command: bundle exec rails s -p 3000 -b '0.0.0.0'
+    volumes:
+        - .:/workspace
+    ports:
+        - "3000"
```
Comment (grogy) [inline docker-compose.yml]: There is necessary `- "3000:3000"`.

It was tested in Ubuntu 16.04LTS, kernel 4.4, Docker 1.13.1.

When do you change it, then we can merge it.

Comment (vsobotka) [reply docker-compose.yml]: Do you get an error? Because it works as is.

Docker version 1.13.1
docker-compose version 1.5.2
Ubuntu 16.10, kernel 4.8

**Comment by grogy** [comment]
It works  for me now.

```console
$ uname -a
Linux engineer-notebook 4.4.0-62-generic #83-Ubuntu SMP Wed Jan 18 14:10:15 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

$ docker info
Containers: 32
 Running: 0
 Paused: 0
 Stopped: 32
Images: 248
Server Version: 1.13.1
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 416
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: aa8187dbd3b7ad67d8e5e3a15115d3eef43a7ed1
runc version: 9df8b306d01f59d3a8029be411de015b7304dd8f
init version: 949e6fa
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.4.0-62-generic
Operating System: Ubuntu 16.04.2 LTS
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 7.681 GiB
Name: engineer-notebook
ID: NGEM:HM5C:HF2S:MGNT:5QKL:XPN2:GHEV:K2SM:25JG:UZH5:3ATN:6O6W
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Comment by vsobotka** [comment]
I wanted the "3000" option because it won't collide with other possible containers on your machine. If only a number of exposed port is specified, random host port will be chosen. As stated in the docs:
https://docs.docker.com/compose/compose-file/#/ports

Also, it looks like no change in this regard has been made in older versions of `docker-compose`. No idea what the problem is.

**Pull request merged**